pages/api/contract-metadata.ts

```diff
@@ -15,9 +15,8 @@ export default function handler(
   res.status(200).json({
     name: "Long Take NFT Original",
     description: `\
-Long Take NFTs are video NFTs up to 10GB in size that are transcoded to optimally playback for viewers on any device or bandwidth.
-
-Powered by Livepeer.`,
+Long Take NFTs are video NFTs up to 10GB in size that  are transcoded on the Livepeer network to optimally playback for viewers on any device or bandwidth.
```

Comment:
There is an extra space between `that` and `are`, other than that it looks good.

Reply:
Changes made